includes/gateways/stripe/includes/admin/admin-actions.php

```diff
@@ -388,3 +458,102 @@ function give_stripe_show_currency_notice() {
 }
 
 add_action( 'admin_notices', 'give_stripe_show_currency_notice' );
+
+/**
+ * Disconnect Stripe Account.
+ *
+ * @since 2.7.0
+ *
+ * @return void
+ */
+function give_stripe_disconnect_connect_stripe_account() {
+
+	$get_data = give_clean( $_GET );
+
+	if ( is_admin() && isset( $get_data['stripe_disconnected'] ) ) {
+		$account_name = ! empty( $get_data['account_name'] ) ? $get_data['account_name'] : false;
+
+		// Disconnect Stripe Account.
+		give_stripe_disconnect_account( $account_name );
+	}
+}
+
+add_action( 'admin_init', 'give_stripe_disconnect_connect_stripe_account' );
+
+/**
+ * Set default Stripe account.
+ *
+ * @since 2.7.0
+ *
+ * @return void
+ */
+function give_stripe_set_account_default() {
+
+	$post_data    = give_clean( $_POST );
+	$account_slug = ! empty( $post_data['account_slug'] ) ? give_clean( $post_data['account_slug'] ) : false;
```

Comment:
@mehul0810 remove extra function call for `give_clean`.
```suggestion
	$account_slug = ! empty( $post_data['account_slug'] ) ? $post_data['account_slug'] : false;
```